daisy_workflows/image_import/suse/translate.py

```diff
@@ -25,98 +24,126 @@
 import re
 
 import utils
+import utils.configs as configs
 import utils.diskutils as diskutils
 
 
 class _Package:
   """A Zypper package to be installed.
 
   Attributes:
-      name (str): Zypper's name for the package.
-      gce (bool): Is the package specific to GCP? When set to true, this
-                  package will only be installed if when `install_gce_packages`
-                  is also true.
+      name: Zypper's name for the package.
+      gce: Is the package specific to GCP? When set to true, this
```

Comment:
Any reason for removing types from the doc string?

Reply:
Good question -- I was reading go/python-tips/035, and it encouraged using PEP-484 annotations instead of docstring annotations.